src/testFixtures/workspaceFolder/ruby-plain-sam-app/src/app.rb

```diff
@@ -0,0 +1,3 @@
+def handler_beside_package_json(event, context)
```

Comment:
nit: This test file doesn't have any imports, I would suggest adding some import statements so that you can properly test the output of `searchDependency`

Reply:
Added Import statement to **app.rb** file